src/SonarQube.Common/AnalysisConfig/AnalysisConfigExtensions.cs

```diff
@@ -35,6 +35,8 @@ public static class ConfigSettingsExtensions
         /// </summary>
         private const string SettingsFileKey = "settings.file.path";
 
+        public const string IsUsingCommandLineCredentialsKey = "begin.hasCredentials";
```

Comment:
Wasn't sure where to specify this constant so I put it next to the one used in the same property bag.